app.rb

```diff
@@ -65,9 +66,16 @@ def list_all_labels
     end
   end
 
+  # List all authors option "6"
   def list_all_authors
-    # add code here
-    puts 'List all authors'
+    puts 'List of all authors'
+    if @authors.length <= 0
+      puts 'No games to show'
```

Comment:
Please change "No games.." to "No authors"

Reply:
Hi @SergSkol,
Thank you for notifying.
Changed the same.